### PR TITLE
BAEL-6399: Getting Yesterday's Date in Java

### PR DESCRIPTION
**Pull request by azhwani**
Simple PR

### CODE REVIEW

File: core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.joda.time.Instant;
+import org.junit.jupiter.api.Test;
+
+class GetYesterdayDateUnitTest {
+
+    @Test
+    void givenDate_whenUsingCalendarClass_thenReturnYesterday() {
+        // Dummy date
```
Comment (vinipx) [inline core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java:18]: you can remove all the comments from here and all remaining test methods - the local variables are already explicit enough :)

File: core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import java.util.GregorianCalendar;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.joda.time.Instant;
+import org.junit.jupiter.api.Test;
+
+class GetYesterdayDateUnitTest {
+
+    @Test
+    void givenDate_whenUsingCalendarClass_thenReturnYesterday() {
```
Comment (vinipx) [inline core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java:17]: I would keep the order of test methods in alignment with the article's sections

**Review by vinipx — CHANGES_REQUESTED** [review]

File: core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java (hunk 1 above)
Comment (azhwani) [reply core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java]: Done

File: core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java (hunk 2 above)
Comment (azhwani) [reply core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java]: Done

**Pull request merged**